data_portal/viewsets.py

```diff
@@ -521,3 +524,18 @@ def by_samples(self, request):
             tn_pairing.add_sample(sample_id=sample_id)
         tn_pairing.by_samples()
         return Response(data=tn_pairing.job_list)
+
+
+class SequenceViewSet(ReadOnlyModelViewSet):
+    serializer_class = SequenceSerializer
+    pagination_class = StandardResultsSetPagination
+    filter_backends = [filters.OrderingFilter, filters.SearchFilter]
+    ordering_fields = SEQUENCE_SEARCH_FIELDS
```

Comment:
This can simply replace with `ordering_fields = '__all__'` and remove `SEQUENCE_SEARCH_FIELDS` var.